packages/nimble-components/src/rich-text/editor/index.ts

```diff
@@ -393,10 +427,17 @@ export class RichTextEditor extends FoundationElement implements ErrorPattern {
             // See: https://github.com/ni/nimble/issues/1527
             inclusive: false,
             parseHTML() {
-                // To load the `nimble-anchor` from the HTML parsed content by markdown-parser as links in the
-                // Tiptap editor, the `parseHTML` of Link extension should return `anchorTag`. This is because the
-                // link mark schema in `markdown-parser.ts` file uses `<nimble-anchor>` as anchor tag and not `<a>`.
-                return [{ tag: anchorTag }];
+                // To load the `nimble-anchor` from the HTML parsed content by markdown-parser as links in the Tiptap editor, the `parseHTML`
+                // of Link extension should return nimble `anchorTag` in addition to `<a>` tag as it is used when pasting a link from external
+                // source. This is because the link mark schema in `markdown-parser.ts` file uses `<nimble-anchor>` as anchor tag and not `<a>`.
```

Comment:
Did you meant? 

```suggestion
                // To load the `nimble-anchor` from the HTML parsed content by markdown-parser as links in the Tiptap editor, the `parseHTML`
                // of Link extension should return nimble `anchorTag`.
                // This is because the link mark schema in `markdown-parser.ts` file uses `<nimble-anchor>` as anchor tag and not `<a>`. 
                // ---
                // `<a>` tag is added here to support when pasting a link from external
                // source.
```

Reply:
Yes, I think the suggested change is more clear.